src/core/forms.py

```diff
@@ -377,13 +379,38 @@ def clean_email(self):
 
 
 class LoginForm(forms.Form):
-    user_name = forms.CharField(max_length=255)
-    user_pass = forms.CharField(max_length=255, widget=forms.PasswordInput)
+    user_name = forms.CharField(max_length=255, label="Email")
```

Comment:
Can we make the label a translatable with ugettext?

Reply:
I left the translation logic in the templates as that is what was the current behaviour, but happy to change that as well if you want

Reply:
Woops, missed that. Its fine as is I think.